<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductsForm</class>
 <widget class="QWidget" name="ProductsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="products_stacked_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>661</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Descripción:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="description_text_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cantidad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="stock_spinbox">
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Precio:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="price_dspinbox">
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Categoría:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="category_combo_box"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="update_button">
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_button">
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_view_products">
        <property name="text">
         <string>Ver productos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>20</y>
       <width>611</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_by_id_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Categoría:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_by_category_combo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search_by_name_line_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="refresh_table_button">
          <property name="text">
           <string>Recargar Tabla</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="new_product_button">
          <property name="text">
           <string>Agregar Producto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="products_table"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
